routes/clients.rb

```diff
@@ -80,15 +80,25 @@ def self.registered(app)
                       contact_email: params[:contact_email] || client.contact_email,
                       contact_name: params[:contact_name] || client.contact_name,
                       active: params[:active] || client.active)
-        redirect "/clients/#{params[:id]}"
-      end
 
-      app.delete '/clients/:id/?' do
```

Comment:
I discovered that the app.delete was never being called. There is a way to call the delete method, https://gist.github.com/victorwhy/45bb5637cd3e7e879ace but I was not successful in getting the Post and Delete to work simultaneously on the same form.

I thought it would be easier just to have the app.patch check if the active state was changed, then either keep or delete the resource accordingly.